core-plugins/pom.xml

```diff
@@ -222,6 +222,7 @@
               co.cask.hydrator.plugin.validator.*;
               org.apache.avro.mapreduce;
               parquet.avro.*;
+              parquet.hadoop.*;
```

Comment:
Why are we adding this dependency? In https://github.com/caskdata/hydrator-plugins/pull/257, we removed it because it was causing SparkPluginsTest to fail.